gateway/ln-gateway/src/lib.rs

```diff
@@ -1130,7 +1130,7 @@ impl Gateway {
                                     }
                                 }
                             } else {
-                                warn!("GatewayState must be Running to register with federation. Current state: {gateway_state:?}");
+                                info!("Will not register federation yet because gateway still not in Running state. Current state: {gateway_state:?}. Will keep waiting...");
```

Comment:
I think this is the reason why on startup the gateway doesn't register. At least I also observed this warning after restart. I'm ok with this patch, but would like to see if we can do something to retry sooner in that case.

Reply:
Nice catch. I didn't connect the two problems. Changed to retry with a smaller delay.